netlify.toml

```diff
@@ -2326,6 +2326,11 @@ to = "/podcast/episode/177-stream-processing-kafka-die-basis-moderner-datenpipel
 status = 301
 force = true
 
+[[redirects]]
+from = "/roll-up"
```

Comment:
if you can still change it, I would prefer `alps-roll-up`

Reply:
😬 to late it is already printed :/

Reply:
theeen let's merge